include/deal.II/base/cuda_size.h

```diff
@@ -38,6 +38,19 @@ namespace CUDAWrappers
    * Define the number of threads in a warp.
    */
   constexpr int warp_size = 32;
+
+  /**
+   * Define the largest finite element degree that can be solved using
+   * CUDAWrappers::MatrixFree. Changing this number will affect the amount of
+   * constant memory being used.
+   */
+  constexpr unsigned int mf_max_elem_degree = 10;
```

Comment:
Is there an assert somewhere if somebody goes above this? I couldn't see it in this PR.

Reply:
The `cudaMemcpyToSymbol` will return an error code that we assert